src/pages/settings/Preferences/PreferencesPage.js

```diff
@@ -1,24 +1,25 @@
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
 import React from 'react';
-import {View, ScrollView} from 'react-native';
+import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
 import PropTypes from 'prop-types';
-import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
 import Navigation from '../../../libs/Navigation/Navigation';
 import ROUTES from '../../../ROUTES';
 import ONYXKEYS from '../../../ONYXKEYS';
 import styles from '../../../styles/styles';
+import themeColors from '../../../styles/themes/default';
 import Text from '../../../components/Text';
 import CONST from '../../../CONST';
 import * as User from '../../../libs/actions/User';
-import ScreenWrapper from '../../../components/ScreenWrapper';
 import Switch from '../../../components/Switch';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
 import compose from '../../../libs/compose';
 import withEnvironment, {environmentPropTypes} from '../../../components/withEnvironment';
 import TestToolMenu from '../../../components/TestToolMenu';
 import MenuItemWithTopDescription from '../../../components/MenuItemWithTopDescription';
+import IllustratedHeaderPageLayout from '../../../components/IllustratedHeaderPageLayout';
+import PreferencesDJAnimation from '../../../../assets/animations/PreferencesDJ.json';
```

Comment:
Should we be importing these directly or port them through a file similar to how we handle illustrations/icons with Expensicons? That way if we reuse animations and need to update the file name or something we could do it in a single file